tritium-lib/src/main/java/com/palantir/tritium/proxy/ByteBuddyInstrumentation.java

```diff
@@ -334,59 +323,10 @@ public boolean isAlive() {
         }
     }
 
-    private enum LoggerInitializer implements LoadedTypeInitializer {
```

Comment:
Why don't we need this any longer? Is this because the injected advice is not using the logger anymore?

Reply:
The logger is encapsulated within the new `Handlers` utility class. It was used when a pre/onSuccess/onFailure handler interaction threw an exception, but the generated bytecode isn't responsible for that anymore.